README.md

```diff
@@ -1 +1,23 @@
-# hw01
\ No newline at end of file
+# hw01
+
+![](https://github.com/jchon11/myrepo/blob/master/joshchon.png)
```

Comment:
Good use of a link to get images! You can also download an image, put it in the directory you're Rmd is in, and use a relative path to render. See [here](https://www.earthdatascience.org/courses/earth-analytics/document-your-science/add-images-to-rmarkdown-report/) for more.